backend/kesaseteli/applications/jobs/daily/tests/test_clean_youth_applications.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+from dateutil.relativedelta import relativedelta
+from django.utils import timezone
+
+from applications.jobs.daily import (
+    clean_inactive_youth_applications,
+    clean_old_youth_applications,
+    clean_rejected_youth_applications,
+)
+from applications.models import YouthApplication
+from common.tests.factories import (
+    AcceptedYouthApplicationFactory,
+    ActiveYouthApplicationFactory,
+    InactiveYouthApplicationFactory,
+    RejectableYouthApplicationFactory,
+    RejectedYouthApplicationFactory,
+    YouthApplicationFactory,
+)
+
+
+def setCreationDate(applications, created_at):
```

Comment:
This should be named `set_creation_date`, see [PEP8 naming conventions](https://peps.python.org/pep-0008/#function-and-variable-names)

Reply:
Just wonder why the linter does not complain about it? :)